chain/jsonrpc/tests/rpc_query.rs

```diff
@@ -155,6 +156,32 @@ fn test_status() {
     .unwrap();
 }
 
+/// Retrieve client status failed.
+#[test]
+fn test_status_fail() {
+    init_test_logger();
+
+    System::run(|| {
+        let (_, addr) = start_all(false);
+
+        let mut client = new_client(&format!("http://{}", addr));
+        WaitOrTimeout::new(
+            Box::new(move |_| {
+                actix::spawn(client.status().then(|res| {
+                    if res.is_err() {
+                        System::current().stop();
+                    }
+                    future::result(Ok(()))
+                }));
+            }),
+            100,
```

Comment:
better use named constants instead of magic numbers?

Reply:
this is in tests.
also in clion we actually see what are those numbers mean

Reply:
in GitHub we don't and that's where most of code reading happens